TonerWindow.cpp

```diff
@@ -10,14 +10,43 @@
 */
 
 #include "TonerWindow.h"
-#include "TonerView.h"
+#include "TonePlayer.h"
 
+#include <Alert.h>
 #include <Application.h>
+#include <LayoutBuilder.h>
+#include <Menu.h>
+#include <MenuField.h>
+#include <MenuItem.h>
+#include <PopUpMenu.h>
 
-TonerWindow::TonerWindow() : BWindow(BRect(100, 100, 220, 170), "Toner v1.0", B_TITLED_WINDOW, B_NOT_RESIZABLE | B_WILL_ACCEPT_FIRST_CLICK)
+enum 
 {
-	TonerView *theView = new TonerView( Bounds() );
-	AddChild(theView);
+	Cmd_Level	= 'cm00',
+	Cmd_Freq,
+};
+
+TonerWindow::TonerWindow() : BWindow(BRect(100, 100, 250, 180), "Toner v1.0", B_TITLED_WINDOW, B_NOT_RESIZABLE | B_WILL_ACCEPT_FIRST_CLICK |  B_AUTO_UPDATE_SIZE_LIMITS)
```

Comment:
While Tone does not adhere to Haiku's coding style, these very long lines are abit awkward. How about adding a linebreak?
Also, let's remove the "v1.0" from the window title.